integration/networktest/env/network_setup.go

```diff
@@ -0,0 +1,32 @@
+package env
```

Comment:
Removing the `testnetEnv` can simplify things. Does not seem like it's reused and if the`networktest.Environment` interface changes the `testnetEnv` and the underlying `testnetConnector` have to be changed.

Would prolly mean that the `network_setup.go` file would be removed and files could pattern off with `dev_network.go` and `testnet_network.go` ?

Reply:
It's definitely ugly under the hood but I think it's worth if for the interface personally. The environment abstraction makes it easy to change what you are running a test against. I think `env.LocalDevNetwork()` helps with that usage too but we can discuss.

Reply:
Can still use the same approach and maintain the same methods `env.Testnet()`, `env.DevTestnet()` `env.LocalDevNetwork`.
It's the middle layer `testnetEnv` that's not really needed nor reused. It wraps a `testnetConnector`.
Both `Testnet` `DevTestnet` could just return a compatible `networktest.Environment` struct.
I might be missing something tho, chat when you have tick :) ?

Reply:
"could just return a compatible networktest.Environment struct" is that not exactly what testnetEnv is? It's just a wrapper to make the testnet connector compatible with the Environment interface

Reply:
Yeah exactly. I might be missing something, but why wrap instead of making testnet connector compatible with the  networktest.Environment  interface ? ( I think it would be adding the Prepare method ? )